plugin/trino-mysql/src/test/java/io/trino/plugin/mysql/TestMySqlTypeMapping.java

```diff
@@ -1005,6 +1005,17 @@ public void testUnsignedTypes()
                 .execute(getQueryRunner(), mysqlCreateAndInsert("tpch.mysql_test_unsigned"));
     }
 
+    @Test
+    public void testEnumTypes()
```

Comment:
nit: Rename to `testEnum`. The reason why above method `testUnsignedTypes` uses `Types` suffix is it contains several types in a test.